modules/custom/az_core/az_core.module

```diff
@@ -529,3 +529,23 @@ function az_core_preprocess_container__text_format_filter_help(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_entity_embed_container().
+ *
+ * Prevents competing submit button lockup errors in edit pages
```

Comment:
```suggestion
 * Prevents completing submit button lockup errors in edit pages
```

Reply:
I think it's actually meant to `competing`, not `completing`, i.e. errors from multiple submit buttons on the page "competing" with each other (which happened with embedded webforms when editing a page).

Reply:
Ah 🤦‍♀️ That makes much more sense. Sorry!